pymyq/request.py

```diff
@@ -109,13 +113,19 @@ async def _send_request(
         json: dict = None,
         allow_redirects: bool = False,
     ) -> Optional[ClientResponse]:
-
+        if self._block_request_until is not None:
+            if datetime.utcnow() >= self._block_request_until:
+                self._block_request_until = None
+            else:
+                _LOGGER.debug("Skipping sending request because we are currently timed out.")
+                raise UserRateLimit("Timed out until %s", str(self._block_request_until))
```

Comment:
This doesn't format correctly in the log.

Unexpected error fetching myq devices data: ('Timed out until %s', '2023-09-30 02:20:30.181724')

Reply:
You'll need to use f-strings
```suggestion
                raise UserRateLimit(f"Timed out until {str(self._block_request_until)}")
```

Or %-formatting
```suggestion
                raise UserRateLimit("Timed out until %s" % str(self._block_request_until))
```

Reply:
Yep - I had them as logs originally on my side then changed to an error. I have it fixed locally on my side with a few other things i'm looking at